### DISCUSSION TITLE
September 19th Community Call [alternate time zone call]

### DISCUSSION THREAD

**Discussion by SydneyhSmith** [PowerShell/PowerShell#24250] [Community Call]
This call will be at 7pm PST to accommodate alternate time zones than the regular 9:30am PST call

- PlatyPS update (Jason)
- PSResourceGet release (Sydney)
- Group Policy issue  (Sydney) 
- PowerShell 7.5 / 7.2 EOL (Michael)
- MiniCon (Sydney)
- DSC (Steve)
- WAM feedback (Alex)
- Demo (Jordan)

Please add questions/demos/topics

**Comment by glennsarti** [comment]
Hi @SydneyhSmith, do you know where I can get the call details? I assume it's a Team meeting somewhere ...

**Reply by jborean93** [reply]
Some more details are at https://github.com/PowerShell/PowerShell-RFC/blob/master/CommunityCall/README.md.

**Comment by jborean93** [comment]
If there's time would love to do a quick demo of a new provider module for interacting with GitHub gists https://github.com/jborean93/PowerShell-GistHub. I did do a demo last time so happy to prioritise others before me.

**Reply by SydneyhSmith** [reply]
That would be great!

**Reply by jborean93** [reply]
Hi @SydneyhSmith unfortunately something has come up and I'm not able to make the meeting so won't be able to do this demo. Sorry for the mess around.

**Reply by SydneyhSmith** [reply]
No worries, we will catch you next time

**Comment by StevenBucher98** [comment]
@SteveL-MSFT to demo DSCv3

**Comment by o-l-a-v** [comment]
Questions:

* **PSResourceGet:** User context out of OneDrive for Business Known Folder Move, when? It's a highly requested feature to add the ability to override the default install path, especially for `-Scope CurrentUser`.
  * Tell us what you think about the proposal and how you'd like it solved, the community can come up with a PR if you're low on resources.
  * My open PR waiting for feedback, opened 2024-07-13: <https://github.com/PowerShell/PSResourceGet/pull/1673>
  * Some related issues:
    * 2018-10-18 <https://github.com/PowerShell/PowerShell/issues/8069>
    * 2021-06-09 <https://github.com/PowerShell/PowerShell/issues/15552>
    * 2022-04-07 <https://github.com/PowerShell/PSResourceGet/issues/627>
    * 2023-01-12 <https://github.com/PowerShell/PSResourceGet/issues/889>
    * 2023-11-16 <https://github.com/PowerShell/PSResourceGet/issues/1494>
    * 2024-06-08 <https://github.com/PowerShell/PSResourceGet/issues/1661>
* **PowerShell Gallery**: ACR (Azure Container Registry)
  * What's the status on moving Microsoft modules to a public ACR?
  * Is it still happening?
  * Is there a public beta/preview ACR instance?

**Comment by Gijsreyn** [comment]
@SteveL-MSFT Hey Steve, I couldn't attend last Thursday to the PowerShell community call. However, I watched the YouTube video, and it was cool to see you working on the Azure Baseline. Was it a correct assumption that you were referencing `AzureWindowsBaseline.mof` in the [Azure-Policy](https://github.com/Azure/azure-policy/blob/master/samples/GuestConfiguration/package-samples/configurations/AzureWindowsBaseline/AzureWindowsBaseline.mof) repository?

If I understood you correctly, you were looking for a bit of help from the community to convert the `.mof` file, into a valid DSC V3 configuration document. To help you out, check out the following code small code snippet how you could grab the REG_DWORD values from the `.mof` file and use the `yayaml` module (thanks for Mikey's suggestion on this one) to convert it to YAML:

```powershell
$mofFile = ConvertFrom-DscMof -Path "AzureWindowsBaseline.mof" # from https://github.com/MathieuBuisson/Powershell-Utility/blob/master/ConvertFrom-DscMof/ConvertFrom-DscMof.psm1 change line the datetime on the ordered hashtable

$dwords = $mofFile | Where-Object { $_.Path -ne $Null -and $_.Type -eq 'REG_DWORD' }

$resources = foreach ($dword in $dwords)
{
    [ordered]@{
        name       = $dword.Name
        type       = 'Microsoft.Windows/Registry'
        properties = [ordered]@{
            keyPath   = $dword.Path 
            valueName = $dword.Value 
            valueData = @{
                DWord = [int]$dword.RemediateValue # cast as integer value
            }
        }
    }
}

$config = @{
    '$schema' = 'https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/config/document.json'
    resources = $resources
}
$config | ConvertTo-Yaml -Depth 10
```

**Comment by tristanbarcelon** [comment]
@SteveL-MSFT, are there plans to improve ssh experience by modifying the pscore installation process in linux and eliminating the [tedious enablement step](https://www.thomasmaurer.ch/2019/04/setup-powershell-ssh-remoting-in-powershell-6/) if sshd is detected? This would enable [Copy-Item to work remotely from any machine to a linux target via ssh](https://techcommunity.microsoft.com/blog/itopstalkblog/how-to-copy-files-from-windows-to-linux-using-powershell-remoting/681442)
